application/backup.sh

```diff
@@ -136,6 +150,9 @@ fi
 
 echo "postgresql-backup-restore: backup: Completed";
 
+# Clean up temporary files
+rm -f "/tmp/${DB_NAME}.sql.gz" "/tmp/${DB_NAME}.sql.sha256.gz";
```

Comment:
Good thing the DB_NAME doesn't change. If there is an error leaving the temporary file lying around, this will eventually clean it up after a good run.